### PR TITLE
Bump to Elixir 1.3.2

### PR DESCRIPTION
**Pull request by tgautier**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ sudo: required
 language: elixir
 elixir:
   - 1.2.6
-  - 1.3.0
+  - 1.3.2
 otp_release:
   - 18.3
```
Comment (kdisneur) [inline .travis.yml:7]: Shouldn't we add the OTP release 19.0 also ?

Comment (tgautier) [inline .travis.yml:7]: It should !

**Comment by tgautier** [comment]
Fix done

**Comment by kdisneur** [comment]
👍

**Pull request merged**